packages/ui/src/components/vuestic-components/va-tabs/VaTab/VaTab.vue

```diff
@@ -15,7 +15,7 @@
       class="va-tab__content"
```

Comment:
Shouldn't have transition on mount.
![image](https://user-images.githubusercontent.com/5394573/123816341-e7abfb00-d8ff-11eb-8af7-bb583bc68254.png)

Reply:
fixed it